Calendar/Calendar/DayControl.cs

```diff
@@ -41,12 +45,16 @@ public DateTime Date
                 lblDayOfMonth.Text = _date.ToString("dd");
             }
         }
-
+        /// <summary>
+        ///  Control's color change to same as the background color when clicked on another day.
+        /// </summary>
         public void ClearSelection()
         {
             BackColor = Color.ForestGreen;
         }
-
+        /// <summary>
+        /// Vissible of underline the day with save event.
```

Comment:
To jest trochę niejasne ale to wiadomo trzeba przywyknąć żeby dobrze po angielsku przekazać dobre znaczenie.
Ja bym napisał że updatuje wygląd dnia kiedy zmieni się kolekcja jego eventów.